p5js/BodyPix/BodyPix_Webcam_Parts/sketch.js

```diff
@@ -0,0 +1,38 @@
+
+console.log('bodyPix', ml5.bodyPix);
+
+let bodypix;
+let video;
+let segmentation;
+let img;
+function setup(){
+    createCanvas(320, 240);
+
+    // load up your video
+    video = createCapture(VIDEO);
+    video.size(width, height);
+    // video.hide(); // Hide the video element, and just show the canvas
+    bodypix = ml5.bodyPix(video, modelReady)
+    // background(0);
+}
+
+function modelReady(){
+    console.log('ready!')
+    console.log(bodypix)
+    bodypix.segmentWithParts(video, gotResults)
+}
+
+function gotResults(err, result){
+    if(err){
+        console.log(err)
+        return
+    }
+    segmentation = result;
+    // background(0);
+    image(video, 0,0, width, height)
+    image(segmentation.image, 0, 0, width, height)
+    
+
+    bodypix.segmentWithParts(video, gotResults)
```

Comment:
One thing we did with image classification with a webcam is that if you give it a video element when you create the model then it automatically runs over and over (I think). I'm not sure which is best but we should probably be consistent!

Reply:
@shiffman - Ah I see. I will have a look!

Reply:
Yes! Turns out the video automatically runs and gets the next frame. Thanks for flagging!

Reply:
Fixed with: https://github.com/ml5js/ml5-examples/pull/153/commits/1aa58d0ce1d8ce1364252abe1ace76bb3585cc17